### PR TITLE
zh-translation: /news/releases/1.0.x/announcing-1.0.7/index.md #1534

### PR DESCRIPTION
**Pull request by gorda**
Ref: [ServiceMesher#1534](https://github.com/servicemesher/istio-official-translation/issues/1534)

### CODE REVIEW

File: content/zh/news/releases/1.0.x/announcing-1.0.7/index.md [hunk 1]
```diff
@@ -12,73 +12,75 @@ aliases:
     - /zh/news/announcing-1.0.7
 ---
 
-We're announcing immediate availability of Istio 1.0.7 which contains some important security updates. Please see below for details.
+我们很高兴的宣布 Istio 1.0.7 现已正式发布。下面是更新详情。
```
Comment (ilylia) [inline content/zh/news/releases/1.0.x/announcing-1.0.7/index.md]:
```suggestion
我们很高兴的宣布含有一些重要安全更新的 Istio 1.0.7 现已可用。下面是更新详情。
```

File: content/zh/news/releases/1.0.x/announcing-1.0.7/index.md [hunk 2]
```diff
@@ -12,73 +12,75 @@ aliases:
...
 
-## Security update
+## 安全更新{#security-update}
 
-Two security vulnerabilities have recently been identified in the Envoy proxy
-([CVE 2019-9900](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-9900) and [CVE 2019-9901](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-9901)). The
-vulnerabilities have now been patched in Envoy version 1.9.1, and correspondingly in the Envoy builds
-embedded in Istio 1.1.2 and Istio 1.0.7. Since Envoy is an integral part of Istio, users are advised to update Istio
-immediately to mitigate security risks arising from these vulnerabilities.
+最近在 Envoy 代理中发现了两个安全漏洞 ([CVE 2019-9900](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-9900) 和 [CVE 2019-9901](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-9901))。
```
Comment (ilylia) [inline content/zh/news/releases/1.0.x/announcing-1.0.7/index.md]:
```suggestion
最近在 Envoy 代理中发现了两个安全漏洞（[CVE 2019-9900](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-9900) 和 [CVE 2019-9901](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-9901)）。
```

File: content/zh/news/releases/1.0.x/announcing-1.0.7/index.md [hunk 3]
```diff
@@ -12,73 +12,75 @@ aliases:
...
+此漏洞已在 Envoy 1.9.1 版中被修复，相应地，也对 Istio 1.1.2 和 Istio 1.0.7 内置的 Envoy 进行了修复。
```
Comment (ilylia) [inline content/zh/news/releases/1.0.x/announcing-1.0.7/index.md]:
```suggestion
此漏洞已在 Envoy 1.9.1 版中被修复，相应地，Istio 1.1.2 和 Istio 1.0.7 内置的 Envoy 也已被修复。
```

File: content/zh/news/releases/1.0.x/announcing-1.0.7/index.md [hunk 4]
```diff
@@ -12,73 +12,75 @@ aliases:
...
+根据目前的信息，这只会影响 HTTP/1.1 的流量。
+如果您的网络或配置不是这种结构，那么此漏洞不太可能影响到您。
 
-External authorization and rate limit services can check for NULs in headers. Backend servers might have sufficient logging to detect NULs or unintended
-access; it’s likely that many will simply reject NULs in this scenario via 400 Bad Request, as per RFC 7230.
+基于文件的访问日志记录与 gRPC 访问日志记录一样，使用 `c_str()` 表示 header 值，
+因此扫描 NUL，不会发现通过 Envoy 的访问日志的任何异常。
 
-## Detection of path traversal exploit
+相反，操作员可能会在 Envoy 执行的路由和 `RouteConfiguration` 预期的逻辑之间的日志中寻找不一致之处。
```
Comment (rootsongjc) [inline content/zh/news/releases/1.0.x/announcing-1.0.7/index.md]: 操作员 -> 运维人员

File: content/zh/news/releases/1.0.x/announcing-1.0.7/index.md [hunk 5]
```diff
@@ -12,73 +12,75 @@ aliases:
...
-Envoy’s access logs (whether file-based or gRPC) will contain the unnormalized path, so it is possible to examine these logs to detect suspicious patterns and
-requests that are incongruous with the intended operator configuration intent. In addition, unnormalized paths are available at `ext_authz`, rate limiting
-and backend servers for log inspection.
+外部授权和速率限制服务可以检查 header 中的 NUL。
+后端服务器可能具有足够的日志记录来检测 NUL 或意外访问；
+根据 RFC 7230，在这种情况下，很可能会通过 400 bad request 简单地拒绝 NUL。
+
+## 检测路径遍历漏洞{#detection-of-path-traversal-exploit}
+
+Envoy 的访问日志（基于文件或 gRPC ）将包含非规范化路径，因此可以检查这些日志以检测可疑的模式和与预期的操作员配置意图不一致的请求。
```
Comment (rootsongjc) [inline content/zh/news/releases/1.0.x/announcing-1.0.7/index.md]: 操作员 -> 运维人员

Comment (ilylia) [inline content/zh/news/releases/1.0.x/announcing-1.0.7/index.md]:
```suggestion
Envoy 的访问日志（基于文件或 gRPC）将包含非规范化路径，因此可以检查这些日志以检测可疑的模式和与预期的操作员配置意图不一致的请求。
```

Comment (gorda) [reply content/zh/news/releases/1.0.x/announcing-1.0.7/index.md]: @rootsongjc 感谢大佬 review，已修改

**Pull request merged**